modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/notifications/NotificationsSettingsViewModel.kt

```diff
@@ -143,6 +151,12 @@ internal class NotificationsSettingsViewModel @Inject constructor(
         analyticsTracker.track(AnalyticsEvent.SETTINGS_NOTIFICATIONS_SHOWN)
     }
 
+    internal fun onResumed() {
```

Comment:
Can we avoid such function naming? `ViewModel` doesn't have lifecycle so `onResumed()` doesn't relate to it. I think being explicit with something like `checkNotificationPermission()` is clearer.